Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/PDFScrollingPresentationController.mm

```diff
@@ -392,6 +392,14 @@
     return shouldUseInProcessBackingStore() && (layer == m_contentsLayer || pageIndexForPageBackgroundLayer(layer));
 }
 
+#if ENABLE(RE_DYNAMIC_CONTENT_SCALING)
+bool PDFScrollingPresentationController::layerAllowsDynamicContentScaling(const GraphicsLayer*) const
```

Comment:
I fully expect that making "allows" a lie is going to come back and bite me someday. Maybe we need to rename this to be more explicitly about recording DCS display lists during painting? I'm not sure what exactly to go with.